paper-api/src/main/java/org/bukkit/command/defaults/HelpCommand.java

```diff
@@ -91,7 +91,7 @@ public boolean execute(@NotNull CommandSender sender, @NotNull String currentAli
             .append(Component.text("--------- ", NamedTextColor.YELLOW))
             .append(Component.text("Help: ", NamedTextColor.WHITE))
             .append(Component.text(topic.getName()))
-            .append(Component.empty());
+            .append(Component.space());
         if (page.getTotalPages() > 1) {
             header.append(Component.text(" ("))
```

Comment:
Well, if we are front-loading the space we ought to not include it here then huh.